### ISSUE TITLE
Running `python -mallofplos.plos_corpus` fails.

### ISSUE THREAD

**Issue by egh** [PLOS/allofplos#84]
Hi,

When running `python -mallofplos.plos_corpus` this morning, I received the following error:

```
λ python -mallofplos.plos_corpus
Checking for new articles...
4405 new articles to download.
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 3637/3637 [08:40<00:00,  6.99it/s]

4405 new articles downloaded.
Traceback (most recent call last):
  File "/usr/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/egh/.virtualenvs/allofplos/lib/python3.5/site-packages/allofplos/plos_corpus.py", line 845, in <module>
    main()
  File "/home/egh/.virtualenvs/allofplos/lib/python3.5/site-packages/allofplos/plos_corpus.py", line 841, in main
    plos_network=plos_network)
  File "/home/egh/.virtualenvs/allofplos/lib/python3.5/site-packages/allofplos/plos_corpus.py", line 588, in download_check_and_move
    amended_articles = check_for_amended_articles(directory=tempdir)
  File "/home/egh/.virtualenvs/allofplos/lib/python3.5/site-packages/allofplos/plos_corpus.py", line 353, in check_for_amended_articles
    article = Article.from_filename(article_file)
  File "/home/egh/.virtualenvs/allofplos/lib/python3.5/site-packages/allofplos/article_class.py", line 1272, in from_filename
    return cls(filename_to_doi(filename))
  File "/home/egh/.virtualenvs/allofplos/lib/python3.5/site-packages/allofplos/article_class.py", line 42, in __init__
    self.doi = doi
  File "/home/egh/.virtualenvs/allofplos/lib/python3.5/site-packages/allofplos/article_class.py", line 112, in doi
    raise Exception("Invalid format for PLOS DOI")
Exception: Invalid format for PLOS DOI
```

**Comment by egh** [comment]
```
λ pip freeze 
allofplos==0.10.2
certifi==2018.1.18
chardet==3.0.4
idna==2.6
lxml==4.1.1
peewee==3.1.0
pkg-resources==0.0.0
python-utils==2.3.0
requests==2.18.4
six==1.11.0
tqdm==4.19.5
Unidecode==1.0.22
urllib3==1.22
```

**Comment by mpacer** [comment]
what is the filename that this was failing on?

**Comment by egh** [comment]
I modified `/home/egh/.virtualenvs/allofplos2/local/lib/python3.5/site-packages/allofplos/article_class.py:112` to print the doi:

```
raise Exception("Invalid format for PLOS DOI %s" % (d))
```

and now I see:
```
Exception: Invalid format for PLOS DOI 10.1371/annotation/5/site-packages/allofplos/new_plos_articles/plos
```

**Comment by egh** [comment]
Maybe this is because I am using virtualenvwrapper?

**Comment by eseiver** [comment]
OK, looks like `transformations.filename_to_doi()` isn't working correctly. My guess is it's in
`article = 'annotation/' + (filename.split('.', 4)[2])`
and the way `.split()` runs isn't generic enough and needs to be fixed.

**Comment by egh** [comment]
It works when I don't use `virtualenvwrapper`.

**Comment by eseiver** [comment]
fixed in #87
